### ISSUE TITLE
When searched moving to google 

### ISSUE THREAD

**Issue by Sahillather002** [discuitnet/discuit#15]
![image](https://github.com/discuitnet/discuit/assets/72077931/115048f1-9859-4b38-a870-d12211539879)

![image](https://github.com/discuitnet/discuit/assets/72077931/b277949d-d55c-44e9-a42a-7099715ea38e)

**Comment by previnder** [comment]
Yeah, this is by design. Just a workaround until search is implemented.
